nc_test/tst_def_var_fill.c

```diff
@@ -7,6 +7,7 @@
  * Author: Wei-keng Liao.
  */
 
+#include "config.h"
```

Comment:
I put this change in another current PR #1266 so that it gets into the code base. All test programs must start by including config.h. I also checked the other tests in nc_test and nc_test4 and found a bunch more places where the include was missing, so I fixed those too. Thanks for noticing this.